sources/launcher/Stride.Launcher/ViewModels/NewsPageViewModel.cs

```diff
@@ -63,17 +65,12 @@ private async Task OpenUrl()
         public static async Task<List<NewsPageViewModel>> FetchNewsPages(IViewModelServiceProvider serviceProvider, int maxCount)
         {
             var result = new List<NewsPageViewModel>();
-            var rss = new MemoryStream();
+            MemoryStream rss;
             try
             {
-                WebRequest request = WebRequest.Create(Urls.RssFeed);
-                using (var reponse = await request.GetResponseAsync())
-                {
-                    using (var str = reponse.GetResponseStream())
-                    {
-                        str?.CopyTo(rss);
-                    }
-                }
+                HttpResponseMessage response = await httpClient.GetAsync(Urls.RssFeed);
+                response.EnsureSuccessStatusCode();
+                rss = (MemoryStream)await response.Content.ReadAsStreamAsync();
```

Comment:
You should use `CopyTo` here as well instead of casting the returned stream as you're not guaranteed it will always be a `MemoryStream`

Reply:
Updated in second commit.